src/NuGet.Core/NuGet.ProjectModel/ProjectRestoreMetadata.cs

```diff
@@ -132,6 +134,18 @@ public class ProjectRestoreMetadata : IEquatable<ProjectRestoreMetadata>
 
         public RestoreAuditProperties RestoreAuditProperties { get; set; }
 
+        /// <summary>
+        /// A unified flag to help users manage their SDK warning levels. Example: 9.0.100
```

Comment:
```suggestion
        /// A unified flag to help users manage their SDK warning levels. Example: 9.0.100.
```

If you hover your mouse over the property, Visual Studio will show you the IntelliSense info for the property, and will show you how the XML comment renders.